lib/screens/settings/miscellaneous.dart

```diff
@@ -1,41 +1,36 @@
-
 import 'package:flutter/material.dart';
 import 'package:harmonoid/language/constants.dart';
 
 import 'package:harmonoid/screens/settings/settings.dart';
 import 'package:harmonoid/scripts/configuration.dart';
 import 'package:harmonoid/scripts/states.dart';
 
-
 class MiscellaneousSetting extends StatefulWidget {
-  MiscellaneousSetting({Key key}) : super(key: key);
+  const MiscellaneousSetting({Key key}) : super(key: key);
```

Comment:
Didn't know constructors can be `const` in Dart.
Cool!

Reply:
They're used so the widget don't get rebuilt when a parent widget is rebuilt. Used to increase performance. I recommend to add `const` everywhere possible in your app